src/Extensions/Commutator.bonsai

```diff
@@ -56,121 +63,122 @@
-          <Interval>PT0.333S</Interval>
-        </Builder>
-      </Expression>
-      <Expression xsi:type="Disable">
-        <Builder xsi:type="MulticastSubject">
-          <Name>CommutatorTurns</Name>
-        </Builder>
-      </Expression>
       <Expression xsi:type="Combinator">
-        <Combinator xsi:type="DoubleProperty">
-          <Value>0</Value>
+        <Combinator xsi:type="port:SerialReadLine">
+          <port:PortName>Commutator</port:PortName>
+          <port:NewLine>\n</port:NewLine>
         </Combinator>
       </Expression>
+      <Expression xsi:type="Parse">
+        <Pattern>{"target_turns":%f}</Pattern>
+        <Separator />
+      </Expression>
       <Expression xsi:type="Combinator">
-        <Combinator xsi:type="rx:Take">
-          <rx:Count>1</rx:Count>
+        <Combinator xsi:type="dsp:Difference">
+          <dsp:Order>1</dsp:Order>
         </Combinator>
       </Expression>
+      <Expression xsi:type="rx:Accumulate" />
+      <Expression xsi:type="rx:BehaviorSubject">
+        <Name>AccumulatedCommutatorTurns</Name>
+      </Expression>
       <Expression xsi:type="SubscribeSubject">
-        <Name>CommutatorTurns</Name>
+        <Name>OnixBno055Data</Name>
+      </Expression>
+      <Expression xsi:type="MemberSelector">
+        <Selector>Quaternion</Selector>
+      </Expression>
+      <Expression xsi:type="SubscribeSubject">
+        <Name>RigSchema</Name>
+      </Expression>
+      <Expression xsi:type="MemberSelector">
+        <Selector>OnixCommutator</Selector>
+      </Expression>
+      <Expression xsi:type="MemberSelector">
+        <Selector>AdditionalSettings</Selector>
+      </Expression>
+      <Expression xsi:type="MemberSelector">
+        <Selector>ImuCorrectionSampling</Selector>
       </Expression>
       <Expression xsi:type="Combinator">
-        <Combinator xsi:type="rx:Merge" />
+        <Combinator xsi:type="p1:TimeSpanFromSeconds" />
       </Expression>
-      <Expression xsi:type="rx:Accumulate" />
-      <Expression xsi:type="rx:BehaviorSubject">
-        <Name>AccumulatedCommutatorTurns</Name>
+      <Expression xsi:type="PropertyMapping">
+        <PropertyMappings>
+          <Property Name="Interval" />
+        </PropertyMappings>
+      </Expression>
+      <Expression xsi:type="SubscribeSubject">
+        <Name>RigSchema</Name>
+      </Expression>
+      <Expression xsi:type="MemberSelector">
+        <Selector>OnixCommutator</Selector>
+      </Expression>
+      <Expression xsi:type="MemberSelector">
+        <Selector>AdditionalSettings</Selector>
+      </Expression>
+      <Expression xsi:type="MemberSelector">
+        <Selector>ImuRotationAxis</Selector>
+      </Expression>
+      <Expression xsi:type="PropertyMapping">
+        <PropertyMappings>
+          <Property Name="RotationAxis" Selector="X,Y,Z" />
+        </PropertyMappings>
+      </Expression>
+      <Expression xsi:type="GroupWorkflow">
+        <Name>AutoCommutator</Name>
```

Comment:
There is something fishy here. This either needs to be a IncludeWorkflow reference or you need to get rid of AutoCommutator